bin/build-libbitcoind

```diff
@@ -1,9 +1,50 @@
 #!/bin/bash
 root_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/.."
-cd "${root_dir}"
 options=`cat ${root_dir}/bin/config_options.sh`
+h_and_a_dir=$($root_dir/platform/os.sh h_and_a_dir)
+depends_dir=$($root_dir/platform/os.sh depends_dir)
+os_dir=$(${root_dir}/platform/os.sh osdir)
+host=$(${root_dir}/platform/os.sh host)
+btc_dir="${root_dir}/libbitcoind"
+ext=$($root_dir/platform/os.sh ext)
+artifacts_dir=$($root_dir/platform/os.sh artifacts_dir)
+echo "Using BTC directory: ${btc_dir}"
+
+cd "${root_dir}"
+
+copy_header_files () {
+  if [[ -d "${artifacts_dir}" && -d "${h_and_a_dir}" && -d "${btc_dir}" ]]; then
+    mkdir -p "${artifacts_dir}/include" > /dev/null 2>&1
+    pushd "${root_dir}"
+    find libbitcoind -type f \( -name "*.h" -or -name "*.hpp" -or -name "*.ipp" \) -print0 | xargs -0 -I{} rsync -R {} "${artifacts_dir}"/include
```

Comment:
The script appears to be unresponsive here, however letting it run for awhile, it did complete.